packages/create-mc-app/src/tasks/update-custom-application-config.ts

```diff
@@ -72,8 +72,12 @@ function updateCustomApplicationConfig(options: TCliTaskOptions): ListrTask {
   return {
     title: 'Updating Custom Applications config',
     task: () => {
+      const configFileName =
```

Comment:
We update values in a different config file depending on the type of application we want to create.